app/controllers/members_controller.rb

```diff
@@ -24,6 +26,19 @@ def create
     end
   end
 
+  def verify_email
+    @member = Member.find_by_email_confirmation_token(params[:id])
+    if @member
+      @member.activate_account
+      MemberMailer.welcome_email(@member).deliver_now
+      flash[:success] = 'Welcome to the S\'Up! Your email has been confirmed.'
```

Comment:
You can use double quotes around the string and then a single quote works inside, so `"Welcome to S'Up!"`.